src/main/kotlin/Command/base/Processor.kt

```diff
@@ -0,0 +1,19 @@
+package Command.base
+
+import Command.commands.ChooseIngredientCommand
+import Command.commands.ChooseNoodlesCommand
+import Command.commands.ChooseSauceCommand
+import Command.commands.ShowPriceCommand
+
+class Processor(val cart: Cart) {
+
+  fun process(command : String) : Command {
```

Comment:
The other solution I was thinking is a to add a queue of Commands in Processor so you append new commands and then `process()` would execute all commands and clear the queue. But I didn't want this for the example because I want to make it interchangeable (you can change your base noodles and the cart gets reset) and I want to show the price in any moment too.